app/importers/activity_insight_importer.rb

```diff
@@ -203,13 +203,10 @@ def call
 
           activity_insight_file_location = pub.postprints&.first&.location
 
-          if pub_record.no_open_access_information? && activity_insight_file_location.present? && Publication.oa_publication_types.include?(pub_record.publication_type)
-            aif = pub_record.activity_insight_oa_files.first
+          if activity_insight_file_location.present? && pub_record.can_receive_new_ai_oa_files?
+            aif = pub_record.activity_insight_oa_files.find_by(location: activity_insight_file_location)
 
-            if aif.present?
-              aif.update location: activity_insight_file_location
-              aif.save!
-            else
+            if aif.blank?
               file = ActivityInsightOAFile.create(location: activity_insight_file_location)
               pub_record.activity_insight_oa_files << file
```

Comment:
I think that we need to do [this](https://github.com/psu-libraries/researcher-metadata/blob/47cd1faaa854d865da95fe90480b1a20bded8225/app/services/oa_workflow_service.rb#L6) here also if we're going to queue up the DOI verification job at this point.

Reply:
Ah good catch.  I'll fix that up.